cypress/integration/create.js

```diff
@@ -0,0 +1,29 @@
+describe('Sourcebooks login', function() {
+
+    it('Admin should create a new client', function () {
+
+        const organizationName = Math.ceil(Math.random()*100000);
+
+        cy.visit('/');
+        cy.get('[id="loginForm.userId"]').click({force:true});
+        cy.get('[aria-label="Saulė Raudytė"]').click();
+        cy.get('[id="loginForm.role"]').click({force:true});
+        cy.get('[aria-label="Admin"]').click();
+        cy.get('[type="submit"]').click();
+
+        cy.get('[href="/clients"]').click();
+        cy.get('.btn').click();
+
+        cy.get('[id="clientDetailsForm.organization"]').type(organizationName);
+        cy.get('[id="clientDetailsForm.contacts_firstName_0"]').type(organizationName+1);
```

Comment:
It's always better to leave spaces before and after math operators, it improves code readability :)